src/main/kotlin/net/ccbluex/liquidbounce/utils/render/placement/PlacementRenderHandler.kt

```diff
@@ -126,16 +126,16 @@ class PlacementRenderHandler(private val placementRenderer: PlacementRenderer, v
             return
         }
 
-        scanBlocksInCuboid(2, pos) { // TODO in theory a one block radius should be enough
-            inList.computeIfPresent(it) { _, value ->
-                Triple(value.first, getCullData(it), value.third)
-            }?.let { return@scanBlocksInCuboid false }
```

Comment:
Why did you remove the early return here?

Reply:
The previous code always returns false, and the `scanBlocksInCuboid` will stop if it returns true, so I think it's unnecessary

Reply:
It prevented an unnecessary operation tho.

Reply:
It just prevents the current iteration. I'll add this back. (`return@forEach`)

Reply:
The second return was indeed useless, only the first was useful.